onetbb.natvis

```diff
@@ -0,0 +1,256 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The open question is where this file should be located.
Is it OK to put it in the root directory or we need to create a subdirectory (e.g. `debug_visualizers`, or `debugging/visualizers` and put the file (and any future visualizers for other debuggers) there

Reply:
> The open question is where this file should be located. Is it OK to put it in the root directory or we need to create a subdirectory (e.g. `debug_visualizers`, or `debugging/visualizers` and put the file (and any future visualizers for other debuggers) there

For location -probably it can be integration/visualizers

Reply:
Since this is more Visual Studio related, I am suggesting to use `integration/windows/vs`.

Reply:
Moved to `integration/windows/vs`